internal/service/networkmanager/core_network_policy_document_data_source.go

```diff
@@ -93,6 +93,16 @@ func dataSourceCoreNetworkPolicyDocument() *schema.Resource {
 								},
 							},
 						},
+						"dns_support": {
+							Type:     schema.TypeBool,
+							Default:  true,
+							Optional: true,
+						},
+						"security_group_referencing_support": {
+							Type:     schema.TypeBool,
+							Default:  false,
+							Optional: true,
+						},
```

Comment:
Will these still work as optional/computed with no provider-side default?

Reply:
As these are inputs to a data source that shouldn't be a problem.